nilearn/_utils/tests/test_common.py

```diff
@@ -43,7 +43,7 @@ def test_number_public_functions():
     If this is intentional, then the number should be updated in the test.
     Otherwise it means that the public API of nilearn has changed by mistake.
     """
-    assert len({_[0] for _ in all_functions()}) == 262
+    assert len({_[0] for _ in all_functions()}) == 261
```

Comment:
As `get_vertexcolor` is moved from `html_surface` to `_matplotlib_backend`, the number of public functions descreases. 

Actually it is not used elsewhere, just in `plotting.surface` package. Would that be ok?

Reply:
Then shouldn't we call it `_get_vertexcolor` ?

Reply:
It is in `plotting.surface._matplotlib_backend` package which is private. However it is imported from `plotting.surface.html_surface` and `plotting.surface._plotly_backend` modules. As far as I understand, we name such functions without `_`, non?

Reply:
Although it is not explicitly imported, it is still imported. 

We might think of it as class methods which are only used inside the class and used as public instance methods?